<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstallerWidget</class>
 <widget class="QWidget" name="InstallerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InstallerWidget</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="mainLayout" rowstretch="0,1" columnstretch="1,1">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="openPatchGroupBox">
     <property name="title">
      <string>Open Patch</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="openPatchLayout" rowstretch="0,0">
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>7</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="pathLabel">
        <property name="text">
         <string>Path</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="patchPathEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Patch folder path</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="openPatchButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
        <property name="icon">
         <iconset resource="PatcherResources.qrc">
          <normaloff>:/images/box.svg</normaloff>:/images/box.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="dependenciesGroupBox">
     <property name="title">
      <string>Dependencies</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="dependenciesLayout">
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>7</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="DependenciesListWidget" name="dependenciesListWidget"/>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="checkButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
        <property name="icon">
         <iconset resource="PatcherResources.qrc">
          <normaloff>:/images/test.svg</normaloff>:/images/test.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>17</width>
          <height>17</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="patchListGroupBox">
     <property name="title">
      <string>Patch Objects</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="patchContentLayout">
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>7</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="PatchListWidget" name="patchListWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="installButtonLayout">
        <property name="spacing">
         <number>7</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="installInfoLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Some dependencies not satisfied!</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QPushButton" name="installButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Install</string>
          </property>
          <property name="icon">
           <iconset resource="PatcherResources.qrc">
            <normaloff>:/images/install.svg</normaloff>:/images/install.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>17</width>
            <height>17</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DependenciesListWidget</class>
   <extends>QListWidget</extends>
   <header>DependenciesListWidget.h</header>
  </customwidget>
  <customwidget>
   <class>PatchListWidget</class>
   <extends>QListWidget</extends>
   <header>PatchListWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>patchPathEdit</tabstop>
  <tabstop>openPatchButton</tabstop>
  <tabstop>dependenciesListWidget</tabstop>
  <tabstop>patchListWidget</tabstop>
  <tabstop>checkButton</tabstop>
  <tabstop>installButton</tabstop>
 </tabstops>
 <resources>
  <include location="PatcherResources.qrc"/>
 </resources>
 <connections/>
</ui>
